schedule/jeos/sle/jeos-main.yaml

```diff
@@ -21,6 +21,10 @@ conditional_schedule:
         UEFI:
             '1':
                 - console/verify_efi_mok
+    udev_dep:
+        VERSION:
+            '15-SP4':
```

Comment:
Is there no `>=`? :-/

Reply:
Not supported AFAICT or @jlausuch? Yeah, we need to go back to `main.pm` schedule in case of this test suite. :)

Reply:
no, it's not supported (that's why we moved away from yaml schedules in containers).
I vote to keep it now here and later on move away from the remaining yaml schedules we have, and put everything in main.pm, or even create our `main_jeos.pm` . But that's for a different PR.